db/blob/blob_log_sequential_reader.h

```diff
@@ -10,6 +10,8 @@
 #include "db/blob/blob_log_format.h"
 #include "rocksdb/slice.h"
 
+#define MAX_HEADER_SIZE(a, b, c) (a > b ? (a > c ? a : c) : (b > c ? b : c))
```

Comment:
Would be nice to use `std::max` here but it is unfortunately `constexpr` only since C++14.

Reply:
Could you `#undef` the macro near the bottom of the file just in case?

Reply:
Done  :)